virtualization-formulas/guest/virtualization-guest/init.sls

```diff
@@ -0,0 +1,71 @@
+{%- if pillar.get("tuning", {})["disable_irq_balancing"] %}
+irqbalance_stopped:
+  service.dead:
+    - name: irqbalance
+    - enable: False
+{%- else %}
+irqbalance_started:
+  service.running:
+    - name: irqbalance
+    - enable: True
+{%- endif %}
+
+{%- if pillar.get("tuning", {})["disable_ksm"] %}
+ksm_stopped:
+  service.dead:
+    - name: ksm
+    - enable: False
+
+ksm_disabled:
+  cmd.run:
+    - name: "echo 2 >/sys/kernel/mm/ksm/run"
+    - unless: "test `cat /sys/kernel/mm/ksm/run` -eq 2"
```

Comment:
This and other sysfs entries in the file are not meant to be persistent?

Reply:
For this one I am still unsure since if ksm service is disabled, there should be no need to unshare all existing pages.

Reply:
persisted them